### DISCUSSION TITLE
autofocus in q-input causes inserted text to appear at the end of the line

### DISCUSSION THREAD

**Discussion by perottosson** [quasarframework/quasar#15923] [General - Components / Directives / etc]
Im creating costum input fields where I loop through a fixed number of input fields for display. While this works the input fields looses focus when I write new values. This is solved by adding the autofocus prop. This solves it partly but if I want to insert text the cursor always jumps to the end of the line every time new text is inserted. 

Here is my code:
```
      <div v-for="item in edit.questionAlternatives" :key="item.val">
        <q-input
          v-model="item.val"
          label="New alternative"
          autofocus
        />
      </div>
```

I guess this is caused by the autofocus. 

See the example in the attached gif

Is there a way of solve this?
![example](https://github.com/quasarframework/quasar/assets/24756698/eb64c7c8-e603-42a6-b937-c26fd2c3c132)

**Comment by metalsadman** [comment]
a codepen would help.
